Fix: Remove dangling reference to ExperimentalTFLiteToTosaBytecode

gaikwadrahul8 opened the pull request:
Hi, Team 
This addresses the **NameError** encountered when calling **ExperimentalTFLiteToTosaBytecode()**. The underlying C++ symbol was removed in TensorFlow 2.19 but the Python helper remained leading to a dangling reference. Merged PR https://github.com/tensorflow/tensorflow/pull/83174 This change replaces the function definition with a to provide clearer feedback to users that the function is no longer available.

Fixes: #88053

gaikwadrahul8 committed:
Fix: Remove dangling reference to ExperimentalTFLiteToTosaBytecode

This commit addresses the NameError encountered when calling
.
The underlying C++ symbol was removed in TensorFlow 2.19, but the Python
helper remained, leading to a dangling reference.

This change replaces the function definition with a
to provide clearer feedback to users that the function is no longer available.

Fixes: #88053

gaikwadrahul8 committed:
Fix: Remove trailing blank lines in pywrap_mlir.py

This commit removes the trailing blank lines in
that were causing a  syntax error in the CI/CD pipeline.

This is a follow-up to the previous commit that removed the
 function.

Fixes: #88053

mihaimaruseac commented on tensorflow/python/pywrap_mlir.py:
```diff
@@ -106,24 +106,4 @@ def experimental_run_pass_pipeline(mlir_txt, pass_pipeline, show_debug_info):
-      ordered_input_arrays,
-      ordered_output_arrays,
-  )
+  return ExperimentalWriteBytecode(filename.encode('utf-8'), mlir_txt.encode())
```
This line should not change. Please configure your editor to insert a newline at the end.
```suggestion
  return ExperimentalWriteBytecode(filename.encode('utf-8'), mlir_txt.encode())

```

mihaimaruseac commented on tensorflow/python/pywrap_mlir.py:
```diff
@@ -106,24 +106,4 @@ def experimental_run_pass_pipeline(mlir_txt, pass_pipeline, show_debug_info):
-  return ExperimentalWriteBytecode(filename.encode('utf-8'), mlir_txt.encode())
-
-
-def experimental_tflite_to_tosa_bytecode(
```
You should also check where this function is being called and remove that too.

gaikwadrahul8 committed:
Fix: Remove all references to ExperimentalTFLiteToTosaBytecode

- Remove Python API and related documentation for ExperimentalTFLiteToTosaBytecode, which was left after the C++ symbol was removed in TF 2.19.
- Fixes NameError and cleans up dangling references.

gaikwadrahul8 committed:
Remove ExperimentalTFLiteToTosaBytecode symbol definitions
